app/services/radical_twilio.rb

```diff
@@ -0,0 +1,17 @@
+class RadicalTwilio
+  def self.send_sms(from:, to:, message:)
+    client.messages.create(from: from, to: to, body: message)
+  end
+
+  def self.send_robocall(from:, to:, url:)
+    client.calls.create(from: from, to: to, url: URI::encode(url))
+  end
+
+  def self.client
+    Twilio::REST::Client.new
+  end
+
+  def self.twilio_phone_number
+    Company.main.next_phone_number
```

Comment:
I'm thinking that next_phone_number logic/method should be moved out of the company model into this class where it seems to belong more

Reply:
that method updates the specific instance (https://github.com/radicalbear/rad_common/pull/124/files#diff-9880236d7a1c3cf97ee0e0a023a4713bR92), although i think its safe to say Company.main will always be used